utils/manifest-utils/src/keystore/mod.rs

```diff
@@ -0,0 +1,301 @@
+pub mod golem_keystore;
+pub mod x509_keystore;
+
+use self::{
+    golem_keystore::{GolemKeystore, GolemKeystoreBuilder},
+    x509_keystore::{X509AddParams, X509CertData, X509KeystoreBuilder, X509KeystoreManager},
+};
+use crate::{golem_certificate::GolemCertificate, policy::CertPermissions};
+use itertools::Itertools;
+use std::{
+    collections::HashSet,
+    ffi::OsStr,
+    fs,
+    path::{Path, PathBuf},
+};
+
+pub enum Cert {
+    X509(X509CertData),
+    Golem {
+        id: String,
+        cert: super::golem_certificate::GolemCertificate,
+    },
+}
+
+impl Cert {
+    pub fn id(&self) -> String {
+        match self {
+            Cert::Golem { id, cert: _ } => id.into(),
+            Cert::X509(cert) => cert.id.to_string(),
+        }
+    }
+
+    pub fn not_after(&self) -> String {
```

Comment:
Why it is needed?

Reply:
Originally it was X509 expiration date. Our certificates will not have expiration date?
It has been assumed it is an useful info to display (even for us, because DevOps issues certs for 3 months only ;))